src/components/panes/JoinSubmissionPane.jsx

```diff
@@ -100,14 +114,33 @@ export default class JoinSubmissionPane extends PaneBase {
                 );
             }
 
+            const dataSection = (state == 'pending')? (
+                <div key="data">
+                    <Msg tagName="h3" id="panes.joinSubmission.data.h"/>
+                    <ul className="JoinSubmissionPane-personData">
+                    {this.props.formItem.data.fields.map(fieldName => {
+                        const fieldItem = this.props.fieldTypes.items.find(item => item.data.slug == fieldName);
+
+                        // Exclude JSON fields
+                        if (fieldItem && fieldItem.data.type == 'json') {
+                            return null;
+                        }
+
+                        const label = fieldItem? fieldItem.data.title : this.props.intl.formatMessage({ id: `misc.fields.${fieldName}` });
```

Comment:
Before custom fields are loaded, this results in formatMessage() running on misc.fields.{custom field}, which results in an error being logged to the console. e.g. `[React Intl] Cannot format message: "misc.fields.extra_text", using message id as fallback.`

Reply:
This is "by design". The error is only printed in development mode, and is just a warning to indicate that something wasn't translated correctly, which is not a problem. I'll consider checking against `NATIVE_FIELDS` but I don't want to complicate things unnecessarily.